internal/exec/exec.go

```diff
@@ -25,6 +25,10 @@ const OpenTracingTagTrivial = "graphql.trivial"
 const OpenTracingTagArgsPrefix = "graphql.args."
 const OpenTracingTagError = "graphql.error"
 
+type contextKey string
+
+const GRAPHQL_FIELDS contextKey = "GRAPHQL_FIELD"
```

Comment:
This identifier name does not follow Go style. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms.